Make.inc

```diff
@@ -1,8 +1,15 @@
 # -*- mode: makefile-gmake -*-
-ifneq ($(OS),Windows_NT)
+ifneq ($(OS),WINNT)
```

Comment:
This isn't right - `Windows_NT` is what gmake sets this to. `WINNT` is a julia convention that was made up out of nowhere